client/src/components/EventView/index.js

```diff
@@ -28,18 +50,26 @@ const EventView = () => {
   return (
     <div>
       <AppBar color={eventColor ?? theme1} />
-      <Layout>
-        Event View
-        <Location address="Provo UT" />
+      <Layout><DisplayStle>
+        <RSVPFloating></RSVPFloating></DisplayStle>
         <Description description="The funest party you will ever image. You will have such a blast. Come and have fun in the sun with everyone. Hello keep reading lots of word. Want to make it wrap around to demonstate. Maybe this will do it."></Description>
-      </Layout>
+        <ExtraPadding></ExtraPadding>
+        <Location address="Provo UT" />
+        <ExtraPadding></ExtraPadding>
+        <h1>Who's invited?</h1>
```

Comment:
There's a component called `<InviteesDisplay>` that includes the heading and the list of people.